---
layout: post
title: 【org.apache.http.conn.ConnectTimeoutException Connect to sonatype-download.global.ssl.fastly.net】
---

# nexus error
```
org.apache.http.conn.ConnectTimeoutException: Connect to sonatype-download.global.ssl.fastly.net:443 [sonatype-download.global.ssl.fastly.net/31.13.86.16] failed: connect timed out
https://blog.csdn.net/zhuwei_clark/article/details/105265269
```

# System->Outreache:management->Disable
![](https://img-blog.csdnimg.cn/20200402111501924.png?x-oss-process=image/watermark,type_ZmFuZ3poZW5naGVpdGk,shadow_10,text_aHR0cHM6Ly9ibG9nLmNzZG4ubmV0L3podXdlaV9jbGFyaw==,size_16,color_FFFFFF,t_70)


# 重启nexus机器之后，nexus需要重新启动，客户端docker pull 时 connection refused

### 1.现象
```
[root@centos121 ~]#docker pull docker.ccbjb.com.cn/library/node:14.12.0
Error response from daemon: Get https://docker.ccbjb.com.cn/v2/: dial tcp 192.168.3.124:443: connect: connection refused
```

docker login docker.ccbjb.com.cn 也显示 connection refused
```
[root@centos121 ~]#docker login docker.ccbjb.com.cn
Authenticating with existing credentials...
Login did not succeed, error: Error response from daemon: Get https://docker.ccbjb.com.cn/v2/: dial tcp 192.168.3.124:443: connect: connection refused
Username (admin): admin
Password:
Error response from daemon: Get https://docker.ccbjb.com.cn/v2/: dial tcp 192.168.3.124:443: connect: connection refused
```



